src/scripts.js

```diff
@@ -231,15 +235,31 @@ function createCookbook() {
       <section class="individual-recipe-card">
         <section class="recipe-card" id="id${recipe.id}">
           <img src="${recipe.image}" alt="${recipe.name}" class="recipe-card-image">
-          <section class="favorite-save-btn-container">
-            <button class="heart-btn"><i class="fa fa-heart"></i></button>
-            <button class="save-recipe-btn"><i class="fa fa-bookmark"></i></button>
-          </section>
+            <i class="fa fa-heart heart-btn"></i>
+            <i class="fa fa-bookmark save-recipe-btn"></i>
         </section>
         <h2 class="recipe-card-title">${recipe.name}</h2>
       </section>`
   });
   addEventListenerToRecipeCards();
   displayYellowBookmarks(user.cookbook);
   displayRedHearts(user.cookbook);
+}
+
+function displayHomeView() {
+  cookbookButton.classList.remove('beige');
```

Comment:
Added a way to get back to the 'home' page, which diplays the recipe of hte day.